assets/pages/bundles-landing/components/Bundles.jsx

```diff
@@ -166,18 +172,78 @@ function getPaperAttrs({ intCmp }): PaperBundle {
 
   return Object.assign({}, bundles.paper, { paperCtaLink, paperDigCtaLink });
 
-}
+};
 
-function getDigitalAttrs({ intCmp }): DigitalBundle {
+const getDigitalAttrs = ({ intCmp }): DigitalBundle => {
```

Comment:
Yes, probably it is worth to try to standardise it in order to avoid unnecessary diff in the future. 
I found the following rules for ES6 arrow functions: 
* From [ES6 In Depth: Arrow functions](https://hacks.mozilla.org/2015/06/es6-in-depth-arrow-functions/) :
> In ES6, this hacks mostly go away if you follow these rules:
> Use non-arrow functions for methods that will be called using the object.method() syntax. Those are the functions that will receive a meaningful this value from their caller.
> Use arrow functions for everything else.

From [es6 & beyond book](https://github.com/getify/You-Dont-Know-JS/blob/master/es6%20&%20beyond/fig1.png)